terraform/api.tf

```diff
@@ -20,8 +20,8 @@ module "api" {
   heroku_web_dyno_quantity    = 1
   heroku_worker_dyno_quantity = 1
 
-  django_default_from_email = "admin@api.dandiarchive.org"
-  django_cors_origin_whitelist = ["https://gui.dandiarchive.org"]
+  django_default_from_email          = "admin@api.dandiarchive.org"
+  django_cors_origin_whitelist       = ["https://gui.dandiarchive.org"]
```

Comment:
We should have a CI workflow to lint our terraform code, as this is the "correct" way to format terraform. I filed #118 for this.